### PR TITLE
#45: Make tests work with Java 8

### PR DESCRIPTION
**Pull request by ChrisGreenaway**

### CODE REVIEW

**Comment by lorban** [comment]
You should make your PR's title start with #45 so that it would be obvious to which but it refers. Otherwise it looks good to me.

**Comment by jhouserizer** [comment]
test this please

**Comment by ChrisGreenaway** [comment]
I've tested it locally, but the cloudbees build isn't picking up the pull request properly:
https://ehcache.ci.cloudbees.com/job/sizeof-pull-requests/17/console

**Comment by jhouserizer** [comment]
The originally triggered build, and the one I triggered both failed with this error in the job's console:

ERROR: Couldn't find any revision to build. Verify the repository and branch configuration for this job.

**Comment by akomakom** [comment]
test this please

**Comment by akomakom** [comment]
test this please

**Comment by akomakom** [comment]
PR Job fixed, Jenkins security issues.

File: src/test/java/org/ehcache/sizeof/SizeOfTestValues.java [hunk 1]
```diff
@@ -41,6 +41,9 @@
             case 7:
                 hotspot32Bit.put("deepSizeOf(new ReentrantReadWriteLock())", 112L);
                 break;
+            case 8:
```
Comment (chrisdennis) [inline src/test/java/org/ehcache/sizeof/SizeOfTestValues.java]: I have no objection to using fallthrough here - makes it clearer what is going on imho.

**Pull request merged**